### PR TITLE
Improve DynamicProxyTest

### PR DESCRIPTION
**Pull request by safris**
Improve `DynamicProxyTest`, re #60.

### CODE REVIEW

File: src/test/java/io/opentracing/contrib/jdbc/DynamicProxyTest.java [hunk 1]
```diff
@@ -13,22 +13,41 @@
...
+    public int compare(final Class<?> o1, final Class<?> o2) {
+      return o1.getName().compareTo(o2.getName());
+    }
+  };
+
+  private static void testGetAllInterfaces(final Object obj, final Class<?> ... expecteds) {
+    final Class<?>[] ifaces = DynamicProxy.getAllInterfaces(obj.getClass());
+    Arrays.sort(ifaces, comparator);
+    Arrays.sort(expecteds, comparator);
+    assertEquals(Arrays.toString(ifaces), Arrays.toString(expecteds));
```
Comment (whiskeysierra) [inline src/test/java/io/opentracing/contrib/jdbc/DynamicProxyTest.java]:
```suggestion
    assertArrayEquals(ifaces, expecteds);
```

File: src/test/java/io/opentracing/contrib/jdbc/DynamicProxyTest.java [hunk 2]
```diff
@@ -13,22 +13,41 @@
...
+    for (final Class<?> iface : ifaces)
+      assertTrue(Arrays.stream(ifaces).anyMatch(i -> i == iface));
```
Comment (whiskeysierra) [inline src/test/java/io/opentracing/contrib/jdbc/DynamicProxyTest.java]:
```suggestion
      assertTrue(Arrays.asList(ifaces).contains(iface));
```

Comment (whiskeysierra) [reply src/test/java/io/opentracing/contrib/jdbc/DynamicProxyTest.java]: Although this whole loop feels weird.

Comment (safris) [reply src/test/java/io/opentracing/contrib/jdbc/DynamicProxyTest.java]: Thank you for this comment! You're absolutely right! I just fixed this.

**Pull request merged**